main.lua

```diff
@@ -115,7 +117,7 @@ return require('./init')(function (...)
   if startRepl == nil and not script then startRepl = true end
 
   if script then
-    require(luvi.path.join(uv.cwd(), script))
+    dofile(luvi.path.join(uv.cwd(), script))
```

Comment:
This needs some special handling to maintain compatibility with things like `luvit .` redirecting to `./init.lua`. Currently getting this error when doing `luvit .`:

```
C:\luvit-relative-require>luvit .
Uncaught exception:
cannot open C:\luvit-relative-require: Permission denied
stack traceback:
        [C]: in function 'dofile'
        [string "bundle:main.lua"]:120: in function 'main'
        [string "bundle:init.lua"]:51: in function <[string "bundle:init.lua"]:49>
        [C]: in function 'xpcall'
        [string "bundle:init.lua"]:49: in function <[string "bundle:init.lua"]:20>
```